mri/test/test_optimizer.py

```diff
@@ -84,6 +85,47 @@ def test_reconstruction_fista_fft2(self):
                     np.testing.assert_allclose(x_final, np.fft.ifftshift(
                         fourier_0.adj_op(data_0)))
 
+    def test_reconstruction_pogm_fft2(self):
+        """ Test all the registered transformations.
+        """
+        print("Process test FFT2 POGM::")
+        for image in self.images:
+            fourier = FFT2(samples=convert_mask_to_locations(
+                                            fftshift(self.mask)),
```

Comment:
Can you explain why we have to do this?

Reply:
Well this is something to be fixed, in another PR. FFT class right now works on shifted data, while all other Fourier classes (NFFT, NUFFT etc) works directly on mask. 
I would really like FFT to change its behaviour, but I would not do this in this PR. Maybe I will get an issue up for it. Will resolve this after referencing the issue here

Reply:
Yes definitely don't do it here, but it seems weird to me. I had never noticed but fair enough.

Reply:
Most of the time when you compute your mask you are putting the DC frequency at the middle of the mask, however, the FFT operator does not compute it in the middle but rather in the boundaries, therefore you either need to add an `fftshift` operation directly on the mask or for each `FFT` operations  this was the easiest solution.

Reply:
Nah, you might have come across this even in cartesian example:

https://github.com/CEA-COSMIC/pysap-mri/blob/5bb4d2ffc95997fa180d6057ac7e3618c16e99e0/examples/cartesian_reconstruction.py#L48-L49


So there will surely be a bigger impact of this change, although it is a right change

Reply:
#45 created. Resolving here